tutorials/using-xgboost/src/run.py

```diff
@@ -0,0 +1,73 @@
+# imports
+import mlflow
+import argparse
+
+import pandas as pd
+import xgboost as xgb
+import dask.dataframe as dd
+
+from distributed import Client
+from dask_mpi import initialize
```

Comment:
Nit: Prefer `import dask_mpi` and then use namespaces `dask_mpi.initialize()` - easier to read.

Reply:
went back and forth on that, it's not how horovod is shown in our examples is what I thought I saw